contents/blog/experiments.md

```diff
@@ -0,0 +1,106 @@
+
+All well and good... except this goes against everything you know about using your own product. You find the vertical view congested, hard to parse, and sort of terrible.
+
+Do you blindly trust the data, or your intuition?
+
+Both have issues. Your intuition might be how you see the world, but not necessarily how people who use the product see the world. At the same time, what if there was a bug in the vertical layout implementation, which counted each interaction twice? Thus, the real number could've been 600, instead of 1200, which massively changes your product direction.
+
+I'd recommend neither blindly trusting the data, nor your intuition. Experiments show you what is happening, but can't answer why. The real institutional knowledge comes with answering the why, and building up  an accurate model of who your users are, what they need, and how they interact with your product.
+
+To answer the why, you need to talk through the causes. [Create hypotheses about why](https://neilkakkar.com/Bayes-Theorem-Framework-for-Critical-Thinking.html) this is happening, watch user session recordings, and then make a call about what you want to do.
+
+That is, bring data to conversations, but also talk through causes.
+
+
+## Changing environments can invalidate experiment results
+
+So, you've finally gotten results for the experiment above, and figured out why they're like this. Turns out, the vertical layout promotes interaction as it allows users to see all steps of the funnel in one go, click on the steps that seem surprising, see the persons involved in that step, watch their recordings, etc. etc. While the horizontal is a bit more frustrating to see all this information at a glance, causing faster bounces.
+
+That's a model that keeps on giving, even when things change.
+
+Let's say its now three months in the future, and you've done a design revamp. Horizontal bars are thinner now, while vertical bars are thicker. As a result, horizontal funnels fit in cleanly on screen, while vertical funnels don't.[^4]
+
+You could run an experiment again to find if user preferences have changed, but if your model is right, interactions should start going down, and you can make the call to revert back to a horizontal layout.
+
+[^4]: A bit contrived, but easier to keep this blogpost manageable without explaining the ins and outs of PostHog graphs :). Also, don't tell our designer, they'll scream if they hear of this design revamp.
+
+All this is trying to say is experiment results don't stand the test of time, and reiterates the importance of extracting a useful model out of your experiment results.
+
+
+## Control for control
+
+5. Control groups? -> All experiments are relative.
+  6. Selection bias
+
+TK: I don't remember what this was supposed to be??
+
+Multi armed bandits? -> No, skip.
+
+8. Clinical trials are very different from website changes -->>> hmm condense this. What am I trying to say?
```

Comment:
Hmm, not sure I want to keep this, but can become decent companion to first point. Actually yes, I like that. Peeking, then nature of web products, then the 3 in flow.